src/data/scripts/VayraMergedModPlugin.java

```diff
@@ -755,17 +756,130 @@ public static String stringifyException(Exception ex) {
     private static class MyLunaSettingsListener implements LunaSettingsListener {
 
         private static class LunaConstants {
```

Comment:
consider pulling this class out of here. really, it has grown big enough for it to clutter the codebase now.

Reply:
done.

Reply:
resolving.